### PR TITLE
Please add Burrow

### PR DESCRIPTION
**Pull request by kulikovav**
Burrow is a kafka consumer lag checker

### CODE REVIEW

File: specs/Burrow.spec [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+License:         ASL 2.0
+URL:             https://github.com/linkedin/Burrow
+
+Source0:         https://github.com/linkedin/%{name}/archive/v%{version}.tar.gz
+
+BuildRoot:       %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+BuildRequires:   golang >= 1.9
+
+Requires:        golang
```
Comment (andyone) [inline specs/Burrow.spec]: This tool requires `golang` for usage?

File: specs/Burrow.spec [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+that can send status out via email or HTTP calls to another service.
+
+################################################################################
+
+%prep
+%setup -qn %{name}-%{version}
+
+%build
+export GOPATH=$(pwd)
+go get -v github.com/linkedin/%{name}
```
Comment (andyone) [inline specs/Burrow.spec]: You download the latest version of sources from GitHub. It means build isn't reproducible and it's not a 1.1.0 release at all.

You can add `rpmbuilder` tag for creating source archive with all dependencies using `gopack`. There is the [example](https://github.com/essentialkaos/kaos-repo/blob/master/specs/etcd.spec#L3) from another golang application spec file.

File: specs/Burrow.spec [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      src/github.com/linkedin/%{name}/config/default-http-post.tmpl \
+      src/github.com/linkedin/%{name}/config/default-slack-delete.tmpl \
+      src/github.com/linkedin/%{name}/config/default-slack-post.tmpl .
+
+%install
+rm -rf %{buildroot}
+
+install -dm 755 %{buildroot}%{_bindir}
+
+install -pm 755 src/github.com/linkedin/%{name}/%{name} \
```
Comment (andyone) [inline specs/Burrow.spec]: I think you should change the name from `Burrow` to `burrow`. Using capital letters in names is discouraged.

File: specs/Burrow.spec [hunk 4]
```diff
@@ -0,0 +1,81 @@
+################################################################################
+
+%define  debug_package %{nil}
+
+################################################################################
+
+Summary:         Burrow - Kafka Consumer Lag Checking
```
Comment (andyone) [inline specs/Burrow.spec]: Application name shouldn't be a part of the summary.

**Review by andyone — CHANGES_REQUESTED** [review]

**Comment by kulikovav** [comment]
Thanks, Anton! I updated the spec file. Could you please take a look on it. Now I get a warning from rpmlint about invalid source. Do I need to do something with that or it is fine?

**Comment by andyone** [comment]
> Now I get a warning from rpmlint about invalid source. Do I need to do something with that or it is fine?

It's ok. I highly recommend using [this](https://github.com/essentialkaos/perfecto/blob/master/common/rpmlint.conf) rpmlint config, with disabled useless warnings. Or use [this](https://hub.docker.com/r/essentialkaos/perfecto/tags/) docker image with _perfecto_, where this config already used.

**Review by andyone — APPROVED** [review]

**Comment by andyone** [comment]
Looks good for me now. Merged.

**Comment by andyone** [comment]
@kulikovav `Burrow-1.1.0` has been added to `kaos-testing` repository.

**Pull request merged**